include/BaselineWalkingController/BaselineWalkingController.h

```diff
@@ -8,13 +8,17 @@ namespace mc_tasks
 {
 class CoMTask;
 class OrientationTask;
+
+namespace force
+{
+class FirstOrderImpedanceTask;
```

Comment:
It's a `struct` in mc_rtc (it only avoids a warning)

Reply:
Thanks. Fixed in https://github.com/isri-aist/BaselineWalkingController/pull/11/commits/7c9e2ca9eeb36afd99597245da13318a9062f849